src/ConstraintsProcessor.ts

```diff
@@ -65,14 +65,10 @@ export const processConstraintsPerTable = async (
     tableName: string,
     migrateOnlyData: boolean
 ): Promise<void> => {
-    if (migrateOnlyData) {
```

Comment:
@chamby 
<code>setSequenceValue</code> is necessary, since it handles cases when migrating only data (into predefined schema).
Generally, the <code>IdentityProcessor</code> must implement both <code>setSequenceValue</code> and <code>createSequence</code> functions.

Reply:
@AnatolyFromPerion I can fix this so that `SETVAL()` is called on the identity column. Alternatively, nmig could `TRUNCATE tableName RESTART IDENTITY;` ([documentation](https://www.postgresql.org/docs/current/sql-truncate.html)) on tables before the data import. What is your preference? I looked for `TRUNCATE` in the source code but could not find it. Does data-only migration assume tables are empty?

Reply:
@chamby 
_Does data-only migration assume tables are empty?_ - No, it doesn't.

Seems that <code>SequenceProcessor::setSequenceValue</code> function is what you need.

Please, have a look at line <code>44</code> there.
<code>const seqName: string = `${ tableName }_${ columnName }_seq`;</code>
Please, check how PostgreSQL names sequences behind <code>iIDENTITY</code> columns, then adjust naming pattern above, if needed.